source/addComment.js

```diff
@@ -1,3 +1,29 @@
 $(document).ready(function(){
-  // add your code here
+  $('button').click(function(event) {
+    event.preventDefault();
+    renderForm();
+  })
+  
+  $("input:last-of-type").click(function(event) {
+    event.preventDefault();
+    var commentBody = document.getElementsByName('comment')[0].value
```

Comment:
Sometimes you are using javascript to grab elements and sometimes you are using jQuery. You should stick to using 1.